lib/manageiq/appliance_console/logical_volume_management.rb

```diff
@@ -41,10 +41,19 @@ def setup
     private
 
     def create_partition_to_fill_disk
-      disk.create_partition_table
-      AwesomeSpawn.run!("parted -s #{disk.path} mkpart primary 0% 100%")
-
+      # Check if you need to create a GPT part table or a MSDOS one in base of
+      # max size of partition table
+      max_msdos_ptable_size = 2199023255552
```

Comment:
This feels like it should be a constant in `LinuxAdmin::Disk` rather than a variable here.

Reply:
also prefer `2.terabytes` if active_support is available

Reply:
@carbonin currently not in LinuxAdmin::Disk, should I add it. `2.terabytes` works fine though